prisma/migrations/20230914194400_init/migration.sql

```diff
@@ -1,3 +1,5 @@
+PRAGMA journal_mode = WAL;
```

Comment:
This does make more sense. But based on this it looks like prisma will acquire the exclusive lock whenever it connects: https://github.com/superfly/litefs/issues/425#issuecomment-1907026733

I'm just not 100% certain this will work in the long term. And I'm not 100% certain I know how to check for this.

Calling my friends from Prisma and LiteFS!

@rtbenfield and @benbjohnson 🙏

Could you tell me whether this should work around this issue: https://github.com/superfly/litefs/issues/425#issuecomment-1908720424